src/test/scala/org/broadinstitute/dsde/firecloud/service/DataUseRestrictionTestFixtures.scala

```diff
@@ -13,7 +13,7 @@ object DataUseRestrictionTestFixtures {
   case class DataUseRestriction(
     GRU: Boolean = false,
     HMB: Boolean = false,
-    DS: Seq[String] = Seq.empty[String],
+    DS: Seq[Int] = Seq.empty[Int],
```

Comment:
This will need coordination with GAWB-2758 and GAWB-2769 since they both look at the values stored for this field.